crates/egui/src/data/input.rs

```diff
@@ -336,7 +336,7 @@ pub struct DroppedFile {
     pub last_modified: Option<std::time::SystemTime>,
 
     /// Set by the `eframe` web backend.
-    pub bytes: Option<std::sync::Arc<[u8]>>,
+    pub stream_url: Option<String>,
```

Comment:
I wonder if this should just be a callback instead ("call this to read the data"). One complicating factor is that it is `async`

Reply:
Yes, quite difficult